SUI/SUI/Product/ProductView.swift

```diff
@@ -0,0 +1,178 @@
+// ProductView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран описания товара
+struct ProductView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let chairImageName = "dining"
+        static let chairString = "Chair"
+        static let heartImageName = "heart"
+        static let priceString = "$55"
+        static let buyButton = "Buy now"
+        static let darkPurpleColorName = "darkPurple"
+        static let lightGrayColorName = "lightGray"
+        static let bagImageName = "bag"
+        static let descriptionTitle = "Description"
+        static let tenSpaceNumber: CGFloat = 10
+        static let progressViewText = "Chars ="
+        static let progressViewScoreText = "/ 150"
+        static let charCountNumber = 150
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        ZStack {
+            VStack {
+                navigationBarView
+                Spacer(minLength: 50)
+                VStack(spacing: 30) {
+                    chairImageView
+                    squareView
+                    HStack {
+                        bagImageView
+                        descriptionTextView
+                        Spacer()
+                    }
+                    charProgressView
+                    textEditorView
+                }
+                Spacer(minLength: 130)
+            }
+            .navigationBarBackButtonHidden(true)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @ObservedObject private var keyboard = KeyboardResponder()
+    @StateObject private var chairViewModel = ProductViewModel()
+
+    private var navigationBarView: some View {
+        Rectangle()
+            .fill(Color.clear)
+            .frame(height: 1)
+            .background(
+                LinearGradient(
+                    colors: [.yellow, .red.opacity(0.9)],
+                    startPoint: .topLeading,
+                    endPoint: .bottomTrailing
+                )
+            )
+    }
+
+    private var chairImageView: some View {
+        Image(Constants.chairImageName)
+            .resizable()
+            .foregroundColor(.yellow)
+            .frame(width: 200, height: 200)
+    }
+
+    private var squareView: some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 10)
+                .fill(.white)
+            VStack {
+                HStack {
+                    chairTextView
+                    Spacer()
+                    heartImageView
+                }
+                .padding(.horizontal, 30)
+                HStack {
+                    priceTextView
+                    Spacer()
+                    buyButtonView
+                }
+                .padding(.horizontal, 30)
+                .padding(5)
+            }
+        }
+        .frame(width: UIScreen.main.bounds.width - 60, height: 120)
```

Comment:
```suggestion
        .frame(width: UIScreen.main.bounds.width - 60, height: 120)
```
60 вот это магическое число